### ISSUE TITLE
Is there plan to add pretrained models on PubLayNet dataset for latest VDU model Donut?

### ISSUE THREAD

**Issue by Abhi001vj** [NielsRogge/Transformers-Tutorials#163]
[PubLayNet](https://github.com/ibm-aur-nlp/PubLayNet) is a very useful daatset for overall document understanding with these labels  "Text","Title", "List","Table", "Figure". Is there a plan to add models for this dataset or is there a way we can finetune the `naver-clova-ix/donut-base-finetuned-cord-v2` model on PubLayNet to detect the text, titles and tables. 
Thank you
